web/src/main/java/org/fao/geonet/EncryptorInitializer.java

```diff
@@ -101,7 +115,15 @@ public void init(GeonetworkDataDirectory dataDirectory) throws Exception {
             encryptorPasswordPropFile = getPropertyFromEnv(PASSWORD_KEY, "");
             // Creates a random encryptor password if the password is empty
             if (StringUtils.isEmpty(encryptorPasswordPropFile)) {
-                Log.info(LOG_MODULE, "Generating a random password for the database password encryptor");
+                if (!firstInitialSetupFlag) {
+                    Log.error(LOG_MODULE, String.format(
```

Comment:
it seems working file, but this message is displayed with a new setup, what can be a bit confusing asn it's not really an error in this case.

Reply:
It should not be displayed on initial setup  due to condition `!firstInitialSetupFlag`.  firstInitialSetupFlag is only supposed to be set to true on the migration task which would be the first new setup.

When you say new setup are you talking about a case where you removed the encryptor properties files and it is re-creating a new one? If so than that could be an error because at that point the data in the database should have been encrypted and creating a new encryption key could end up causing issues.

Could you provide steps used to test so that I can reproduce on my local system.

Reply:
I just run with `mvn jetty:run`, removing previously the H2 database in my home folder. The `encryptor.properties` file is empty in that case and the database is created.

Reply:
OK - you are doing a fresh installation. - Yes I agree that we should not see the warning in that case.  The flag would be false because it is not a migration.

Not sure if there is a flag available to identify that this is a new installation/new database creation.  If not then it looks like I will need to look into setting a flag during new database creation to avoid this error.

Reply:
@josegar74  - I have added a change that should resolve this issue.